src/components/activities/MobileActivityPage.tsx

```diff
@@ -16,6 +16,8 @@ import { ManagerPanel } from './ManagerPanel';
 import { ParticipatingOrgChips } from './ParticipatingOrgChips';
 import { ParticipantDialog, RosterPanel, RosterRowCard } from './RosterPanel';
 
+const MOBILE_FOOTER_HEIGHT_PIXELS = 124.5;
```

Comment:
Maybe do this as a calculation summing the two heights so it's clear where the number came from? Or at least note the two heights in a comment.